### DISCUSSION TITLE
Feature Request: Expose

### DISCUSSION THREAD

**Discussion by daugaard47** [leokhoa/laragon#976] [Ideas]
Please add Expose as an alternative to ngrok in the www > share directory.
It's free!
**Repo:**
https://github.com/beyondcode/expose

This would take Laragon to the next level in my opinion.

**Helpful links:**
1. https://beyondco.de/docs/expose/introduction
2. https://pociot.dev/28-introducing-expose-an-easy-to-use-tunneling-service-implemented-in-pure-php

**Comment by leokhoa** [comment]
Thanks @daugaard47.

Expose looks interesting but it is too new. I am afraid of its reliability and stability in the mean time.

**Comment by zeitstein** [comment]
Any updates on this?

I was not able to get Wordpress and Expose to play together completely: one needs a relative url plugin for the non-admin pages and the admin pages always got redirected to the local version.

Thanks!

**Comment by belzaaron** [comment]
Bump!
